receipt_parser/importer.py

```diff
@@ -212,9 +225,9 @@ def main():
         img = grayscale_image(img)
         img = remove_noise(img)
         img = deskew(img)
```

Comment:
Instead of `img[0]` and `img[1]` you can also destructure return parameters like so:

```python
[output, angle] = deskew(img)
```

Reply:
Updated this with last commit, thanks for your input! 👍